### PR TITLE
Add remi-php73

### PR DESCRIPTION
**Pull request by obokaman-com**
I've simply added the necessary changes to enable PHP 7.3 using remi repository

### CODE REVIEW

File: templates/centos6/remi.repo.j2 [hunk 1]
```diff
@@ -51,6 +51,15 @@ enabled={{item['enabled']|default(0)}}
 gpgcheck=1
 gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-remi
 
+[remi-php73]
+name=Les RPM de remi de PHP 7.2 pour Enterprise Linux 6 - $basearch
```
Comment (MiLk) [inline templates/centos6/remi.repo.j2]: The name should mention 7.3

File: templates/centos7/remi.repo.j2 [hunk 2]
```diff
@@ -51,6 +51,15 @@ enabled={{item['enabled']|default(0)}}
 gpgcheck=1
 gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-remi
 
+[remi-php73]
+name=Les RPM de remi de PHP 7.2 pour Enterprise Linux 7 - $basearch
```
Comment (MiLk) [inline templates/centos7/remi.repo.j2]: The name should mention 7.3

**Review by MiLk — COMMENTED** [review]
Thank you for your contribution.
As soon as the comments are addressed, we can make a new release.

**Comment by obokaman-com** [comment]
Thanks @MiLk! Changes applied. 👍

**Review by MiLk — APPROVED** [review]

**Comment by MiLk** [comment]
Published on galaxy as v1.3.2

**Pull request merged**